README.md

```diff
@@ -121,17 +121,24 @@ map.pm.enableDraw('Circle', options);
 map.pm.Draw.getShapes();
 
 // listen to when drawing mode gets enabled
-map.on('pm:drawstart', function(e) {//...});
-
+map.on('pm:drawstart', function(e) {
+    e.shape; // the name of the shape being drawn (i.e. 'Circle')
+    e.workingLayer; // the leaflet layer displayed while editing
```

Comment:
change the doc sentence here to 
`the leaflet layer displayed while drawing`
and we're good to merge 👍 😄